src/biokbase/narrative/jobs/appmanager.py

```diff
@@ -716,9 +715,10 @@ def _build_run_job_params(
             "service_ver": service_ver,
             "params": input_vals,
             "app_id": app_id,
-            "wsid": ws_id,
             "meta": job_meta,
         }
+        if ws_id is not None:
```

Comment:
This is the real change. It makes adding a `ws_id` to the `_build_run_job_params` optional - if it's not present, it doesn't get included in the params at all, instead of just being `None`.